include/klee/util/Bits.h

```diff
@@ -71,13 +71,13 @@ namespace klee {
       assert(N <= 64);
       if (N==0)
         return 0;
-      return ((uint64_t) (int64_t) -1) >> (64 - N);
+      return ((UINT64_C(-1)) >> (64 - N));
```

Comment:
@andreamattavelli Will this still work as before? The previous code casted to a signed integer and then an unsigned integer.